api/src/main/resources/emailTemplate.properties

```diff
@@ -0,0 +1,2 @@
+mail.subject=password reset request
```

Comment:
The file isn't supposed to be a properties file, otherwise you might as well move everything to the messages.properties file which already supports internationalization anyways and get rid of this file.